Signal/src/ViewControllers/DebugUI/DebugUIBackup.m

```diff
@@ -40,6 +40,10 @@ - (nullable OWSTableSection *)sectionForThread:(nullable TSThread *)thread
                                      actionBlock:^{
                                          [DebugUIBackup tryToImportBackup];
                                      }]];
+    [items addObject:[OWSTableItem itemWithTitle:@"Log Database Size Stats"
+                                     actionBlock:^{
+                                         [DebugUIBackup logDatabaseSizeStats];
```

Comment:
I intend to try to shrink our database size on disk/resist its future growth.  This helps.